taichi/common/dict.h

```diff
@@ -14,8 +14,8 @@
 #include <sstream>
 #include <typeinfo>
 
+#include "taichi/common/core.h"
 #include "taichi/math/math.h"
-#include "util.h"
```

Comment:
dict? Move this file to util?

Reply:
As mentioned above `dict` will be removed later.